Java/src/test/java/AirportTest.java

```diff
@@ -1,12 +1,12 @@
-import Planes.experimentalPlane;
+import airport.Airport;
+import planes.ExperimentalPlane;
 import models.ClassificationLevel;
-import models.ExperimentalTypes;
 import models.MilitaryType;
 import org.testng.Assert;
 import org.testng.annotations.Test;
-import Planes.MilitaryPlane;
-import Planes.PassengerPlane;
-import Planes.Plane;
+import planes.MilitaryPlane;
```

Comment:
Please reformat this class

Reply:
Variable was changed to final and renamed

Reply:
please check if "reformat code" action changes something for your class. if not - ignore this thread:
<img width="1633" alt="image" src="https://user-images.githubusercontent.com/90613721/183055885-156a5f7e-e5af-4798-95a5-3a417ff89f8f.png">

Reply:
Done